<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2532</width>
    <height>1740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="9,0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="start_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="status_message_display">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>status_message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="montage_display">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string>montage_display</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,5,1,7,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Module selection</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,5,1,5,1,9">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Estimators</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="MiDaS_Estimator">
              <property name="text">
               <string>MiDaS v3.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="DepthPro_Estimator">
              <property name="text">
               <string>DepthPro</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="VGGT_Estimator">
              <property name="text">
               <string>VGGT</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Object detectors</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="YOLO_Normalized">
              <property name="text">
               <string>YOLO Normalized</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="YOLO_Depth">
              <property name="text">
               <string>YOLO Camera depth</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Other</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QCheckBox" name="NormalizerModule">
              <property name="text">
               <string>Normalizer</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="TrackingModule">
              <property name="text">
               <string>Tracker</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="FrameSaver">
              <property name="text">
               <string>Frame Saver</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="VisualizationModule">
              <property name="text">
               <string>Visualization</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="colormap">
              <property name="text">
               <string>Colormap</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>View selection</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,5,1,5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>RealSense</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="color">
                <property name="text">
                 <string>Color</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="depth">
                <property name="text">
                 <string>Depth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="norm_depth">
                <property name="text">
                 <string>Normalized</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_depth_detections">
                <property name="text">
                 <string>Detection</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_depth_tracked_detections">
                <property name="text">
                 <string>Tracked Detection</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>VGGT</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="vggt_estimated_depth">
                <property name="text">
                 <string>Depth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="norm_vggt">
                <property name="text">
                 <string>Normalized</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_vggt_detections">
                <property name="text">
                 <string>Detection</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_vggt_tracked_detections">
                <property name="text">
                 <string>Tracked Detection</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,5,1,5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>MiDaS</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="midas_estimated_depth">
                <property name="text">
                 <string>Depth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="norm_midas">
                <property name="text">
                 <string>Normalized</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_midas_detections">
                <property name="text">
                 <string>Detection</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_midas_tracked_detections">
                <property name="text">
                 <string>Tracked Detection</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DepthPro</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="pro_estimated_depth">
                <property name="text">
                 <string>Depth</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="norm_pro">
                <property name="text">
                 <string>Normalized</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_pro_detections">
                <property name="text">
                 <string>Detection</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="vis_pro_tracked_detections">
                <property name="text">
                 <string>Tracked Detection</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Save Frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2532</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
